### PR TITLE
script for EB deployments

### PR DESCRIPTION
**Pull request by mattops**

### CODE REVIEW

File: tools/deploy_eb_release.sh [hunk 1]
```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+set -e
+
+VERSION=$3
+ENV=$2
+APP=$1
+DEVTEST_S3_BUCKET="elasticbeanstalk-eu-west-2-429061350814"
```
Comment (iobend3r) [inline tools/deploy_eb_release.sh]: Use config files to store the values like that?

File: tools/deploy_eb_release.sh [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+fi
+
+# check whether user had supplied -h or --help . If yes display usage
+if [[ ( $# == "--help") ||  $# == "-h" ]]
+then
+  display_usage
+  exit 0
+fi
+
+if [[ "$ENV" =~ ^(mock|stage)$ ]]; then
```
Comment (Mjwillis) [inline tools/deploy_eb_release.sh]: only licence uses mock - other apps use dev

Comment (Mjwillis) [reply tools/deploy_eb_release.sh]: So add dev also

File: tools/deploy_eb_release.sh (hunk 1 above)
Comment (mattops) [reply tools/deploy_eb_release.sh]: I'd consider that for a future improvement perhaps but for now I think best to keep it simple.  This script is only going to be used by our team and for the specific apps in the digital studio.

**Pull request merged**